src/stores/Create.ts

```diff
@@ -1,5 +1,4 @@
 import { JsonFormsCore } from '@jsonforms/core';
```

Comment:
That would explain why there was no clean up when removing formstate. I removed it because it kept showing up in searches and annoying me. I'll delete the file.